Formula/node.rb

```diff
@@ -107,12 +88,18 @@ def post_install
       # Dirs must exist first: https://github.com/Homebrew/legacy-homebrew/issues/35969
       mkdir_p HOMEBREW_PREFIX/"share/man/#{man}"
       rm_f Dir[HOMEBREW_PREFIX/"share/man/#{man}/{npm.,npm-,npmrc.,package.json.}*"]
-      ln_sf Dir[libexec/"npm/lib/node_modules/npm/man/#{man}/{npm,package.json}*"], HOMEBREW_PREFIX/"share/man/#{man}"
+      ln_sf Dir[libexec/"npm/man/#{man}/{npm,package.json}*"], HOMEBREW_PREFIX/"share/man/#{man}"
     end
 
     npm_root = node_modules/"npm"
     npmrc = npm_root/"npmrc"
     npmrc.atomic_write("prefix = #{HOMEBREW_PREFIX}\n")
+
+    # This is a slightly annoying hack that let's us keep a perfectly functioning
+    # npm in both HOMEBREW_PREFIX & also privately in libexec.
+    return if File.read(libexec/"npm/bin/npm").include?(opt_libexec)
+    inreplace libexec/"npm/bin/npm", "$basedir/node_modules/npm/bin/npm-cli.js",
```

Comment:
It's possible it's something else breaking local_npm_install_args but the grunt-cli built at https://github.com/Homebrew/homebrew-core/blob/master/Formula/grafana.rb#L35 doesn't work, so something is unhappy.